.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "extends":  [
+    "airbnb-base",
+    "plugin:@typescript-eslint/recommended",
+    "prettier",
+    "prettier/@typescript-eslint"
+  ],
+  "plugins": ["@typescript-eslint", "prettier"],
+  "rules": {
+    "@typescript-eslint/explicit-function-return-type": "off",
+    "@typescript-eslint/explicit-member-accessibility": "off",
```

Comment:
This rule enforces you write `private` or `public` on class methods. On is just the default.